paper.md

```diff
@@ -215,31 +198,31 @@ For example, in the case of Slurm, it currently supports running Docker and Sing
 The container engine plugins abstract generic operations that all engines support such as creating an image from a `Dockerfile`;
 downloading images from a registry and converting them to their internal format;
 and container-level operations such as creation, deletion, and renaming.
-Currently, there are plugins for Docker and Singularity, with others planned by the Popper community.
+Currently, there are plugins for Docker, Podman, and Singularity, with others planned by the Popper community.
 
 The behavior of a resource manager and a container engine can be customized by passing specific configuration through the configuration file.
 This enables the users to take advantage of engine and resource manager specific features in a transparent way.
 In the presence of a `Dockerfile` and a workflow file, a workflow can be reproduced easily in different computing environments only by tweaking the configuration file.
-For example, a workflow developed on the local machine can be run on an HPC cluster using Singularity containers by specifying information about the available MPI library in the configuration file.
+For example, a workflow developed on the local machine can be run on an HPC cluster using Singularity containers by specifying information like the available MPI library, the number of nodes and CPUs to run a job on, etc. in the configuration file.
 The configuration file can be passed through the CLI interface and can be shared among different workflows.
 It can either be created by users or provided by system administrators.
 
 ![Architecture of the Popper workflow engine](./figures/architecture_med.pdf){#fig:arch}
 
-## Continuous Integration
+## Workflow Exporter
 
-Popper allows users to continuously validate their workflows by allowing them to export workflows as CI pipelines for different continuous integration services like Travis, Circle, or Jenkins.
-The tool provides a `ci` subcommand that can be used to generate CI configuration files for different CI services.
-To set up CI for a project using Popper, it is required to generate a CI configuration file, push the project to GitHub and enable the repository on the CI provider.
-Using CI with Popper workflows enhances the reproducibility guarantees as continuous validation helps to keep a check on various breaking changes like outdated dependencies, broken links, or deleted Docker images.
-Another benefit of using CI with Popper is that even without changes, jobs can be configured so that they run periodically (e.g. once a week), to ensure that they are in a healthy state.
+Popper allows exporting workflows to other formats which are more complex in terms of syntax like that of Airflow, Dagster, Travis, GitLab-CI, etc.
```

Comment:
"allows exporting a workflow to other workflow specification formats such as CWL and WDL, as well as those associated with a CI service (e.g. Travis) or workflow engine (e.g. Airflow). In most cases, the workflow specification syntax for these formats is more complex than that one of Popper's, mainly due to the ..."

"Exporting to other formats is done by an extensible _Workflow Exporter_ module that allows to create exporters for an arbitrary list of formats. Currently, plugins for CI services Travis, CircleCI, Jenkins and Gitlab-CI are implemented."